### PR TITLE
feat(component): Card flat and card flat list

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes https://todaydesign.atlassian.net/browse/UOM17-121

[Card Flat](https://deploy-preview-1628--pattern-lib-unimelb.netlify.app/?selectedKind=Today%2FCards&selectedStory=Flat&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Card Flat List](https://deploy-preview-1628--pattern-lib-unimelb.netlify.app/?selectedKind=Today%2FCards&selectedStory=Flat%20List&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [x] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [x] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] Crossbrowser testing (IE11, Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [x] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 2832ab1126021f5b957eebf2f0f20a505a5eff17

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/60a5f7889d122d0007f3e66a

**Review by lmounsey — APPROVED** [review]
Looks good to me! 

I just made a small tweak to responsive styles on the card list. I've also added a new section to documentation to compliment the home page if you guys use it in the future for other Matrix sites

File: components/today/stories/Cards/CardFlatList.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<template>
```
Comment (jaredhenderson) [inline components/today/stories/Cards/CardFlatList.vue:1]: - [ ] Please use realistic example data for stories. Having the exact same heading and description lengths (and image) does not give us an accurate demonstration of this component for assessment.

File: components/cards/CardFlat.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+</template>
+
+<script>
+import ButtonIcon from 'components/buttons/ButtonIcon.vue';
+
+export default {
+  components: {
+    ButtonIcon,
+  },
+  props: {
```
Comment (jaredhenderson) [inline components/cards/CardFlat.vue:41]: - [ ] Are images compulsory for this component (don't appear to be)? What happens to the layout if an image is not supplied? Should there be a story showing this usage?

**Review by jaredhenderson — CHANGES_REQUESTED** [review]

**Review by anormand — APPROVED** [review]
Looks OK and passes testing in both Siteimprove and SortSite

**Comment by jaredhenderson** [comment]
- Looks fine, pending typography colour questions from other card issue (Figma design is using darker fonts for paragraph, etc.)

**Comment by gepifanio** [comment]
@jaredhenderson @lmounsey merging this one, as the colours will be globally changed if needed.

**Pull request merged**